nipype/interfaces/bids.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+""" Set of interfaces that allow interaction with BIDS data. Currently
+    available interfaces are:
+
+    BIDSDataGrabber: Query data from BIDS dataset using pybids grabbids.
+
+    Change directory to provide relative paths for doctests
+    >>> import os
+    >>> import bids
+    >>> filepath = os.path.realpath(os.path.dirname(bids.__file__))
+    >>> datadir = os.path.realpath(os.path.join(filepath, 'grabbids/tests/data/'))
+    >>> os.chdir(datadir)
+
+"""
+
+from .base import (traits,
+                   DynamicTraitedSpec,
+                   BaseInterface,
+                   isdefined,
+                   Str,
+                   Undefined)
+
+try:
+    from bids.grabbids import BIDSLayout
+except ImportError:
+    have_pybids = False
+else:
+    have_pybids = True
+
+from warnings import warn
+
+class BIDSDataGrabberInputSpec(DynamicTraitedSpec):
+    base_dir = traits.Directory(exists=True,
```

Comment:
`Directory` is defined in [`interfaces.traits_extension.py`](https://github.com/nipy/nipype/blob/master/nipype/interfaces/traits_extension.py#L192-L223). Import from `interfaces.base`.